.travis.yml

```diff
@@ -47,7 +48,8 @@ python:
 #
 env:
   matrix:
-    - PYSMT_SOLVER="all"
+    - PYSMT_SOLVER="all" PYSMT_GMPY="TRUE"
+    - PYSMT_SOLVER="all" PYSMT_GMPY="FALSE"
```

Comment:
I I understand this right, we do not test each solver with and without gmpy, but only all the solvers with and without. Right?

Are the single-solver builds with or without gmpy?